.github/workflows/lint-unit.yaml

```diff
@@ -8,11 +8,11 @@ on:
 jobs:
   lint-unit:
     name: Lint and Unit tests
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
If you check the [raw output](https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json), The version 3.6 is there with arch64.

Reply:
Also [issue](https://github.com/actions/setup-python/issues/544) was closed, so this should be resolved soon?
What if we use `runs-on: ubuntu-20.04` with comment for all pythons instead of splitting this job to two different jobs?

Reply:
> If you check the [raw output](https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json), The version 3.6 is there with arch64.

I could not found.
`curl https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json | grep download | grep "python-3.6" | grep "22.04"`

Reply:
You are right I checked only for `python-3.6`.

Reply:
I've changed the base os to 20.04